plugins/modules/gcp_secretmanager_info.py

```diff
@@ -0,0 +1,198 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023, Tze L. <tze@aiyor.net>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import absolute_import, division, print_function
+import os
+import json
+from ansible_collections.google.cloud.plugins.module_utils.gcp_utils import navigate_hash, GcpSession, GcpModule, GcpRequest
+
+__metaclass__ = type
+
+################################################################################
+# Documentation
+################################################################################
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ["preview"], 'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: gcp_secretmanager_info.py
+description:
+- Gather info for GCP Secret Manager
+short_description: Gather info for GCP Secret Manager - List secrets and versions metadata.
+author: Tze L. (https://github.com/tl-aiyor)
+requirements:
+- python >= 2.6
```

Comment:
will this really work with python2.6? Since it's a new module, we could just up the version.

Reply:
What version do you suggest I bump up to? I can amend it if required.

Reply:
I guess ansible 2.14 supports 2.7 or newer as of 2.13: https://endoflife.date/ansible-core#python-compatibility.

So how about updating to 2.7 at least?

Reply:
Updated the req in documentation from 2.6 to 2.7 in #8b88a27